book/src/8.ai/1.chat/2.streaming.md

```diff
@@ -1,28 +1,32 @@
 # Streaming
 
+> [!NOTE]
+> Currently, streaming in Teams is only supported in the personal scope, not group chats or channels.
+
 Now lets add streaming support...
 
 ```typescript
-import { ChatPrompt } from '@microsoft/teams.ai';
-import { OpenAIChatModel } from '@microsoft/teams.openai';
+import { ChatPrompt } from "@microsoft/teams.ai";
+import { OpenAIChatModel } from "@microsoft/teams.openai";
 
 const prompt = new ChatPrompt({
-    instructions: [
-        'you are an assistant that helps find the perfect emoji to use for a given situation.',
-        'you will only respond with emojis.',
-    ].join('\n'),
-    model: new OpenAIChatModel({
-        model: 'gpt-4o',
-        apiKey: process.OPENAI_API_KEY,
-        stream: true,
-    }),
+  instructions: "You are a helpful assistant who talks like a pirate.",
+  model: new OpenAIChatModel({
+    model: "gpt-4o",
+    apiKey: process.env.OPENAI_API_KEY,
+  }),
 });
+```
 
-(async () => {
-    await prompt.chat('having a great day!', (chunk) => {
-        process.stdout.write(chunk); // 😄🌞🎉
-    });
+Somewhere else in your app:
 
-    process.stdout.write('\n');
-})();
+```typescript
+app.on("message", async ({ send, stream, activity }) => {
+  await prompt.send(activity.text, {
+    // Passing in an onChunk handler automatically enables streaming
+    onChunk: (chunk) => {
+      stream.emit(chunk);
```

Comment:
In this case, the lambda saves the stream to be used later?  Is there a 'end of stream' at some point?